src/webapp/azext_webapp/create_util.py

```diff
@@ -91,3 +102,78 @@ def check_app_exists(cmd, rg_name, app_name):
         if item.name == app_name:
             return True
     return False
+
+
+def get_lang_from_content(src_path):
+    # NODE: package.json should exisit in the application root dir
+    # NETCORE: NETCORE.csproj should exist in the root dir
+    runtime_details_dict = dict.fromkeys(['language', 'file_loc', 'default_sku'])
+    package_json_file = os.path.join(src_path, 'package.json')
+    package_netcore_file = os.path.join(src_path, 'netcore.csproj')
```

Comment:
Hang on. This only works if the project is NAME netcore.csproj. It will be *.csproj, not a specific name! This can't ever work, I'm sorry. 

When you type "dotnet new razor" you will get a new project file named after the current folder. @panchagnula

Reply:
Try:

`import glob
os.path.join(src_path, glob.glob("*.csproj")[0])`